.github/workflows/build-push.yml

```diff
@@ -0,0 +1,110 @@
+
+name: Build and Push Docker Image WEAC Codebase
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'python-flask/**'
+      - 'c-sharp-razor/**'
+      - 'rust-actix/**'
+      - 'ruby-sinatra/**'
+      - 'javascript-express/**'
+      - 'go-gin/**'
+      - 'go-gorilla/**'
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
```

Comment:
We should build on alpine:edge right?

Reply:
it is not needed since he is building the docker image that contains the building environment. Or it should have at least